packages/odie-client/src/components/send-message-input/index.tsx

```diff
@@ -1,44 +1,101 @@
 import { HelpCenterSelect } from '@automattic/data-stores';
 import { HELP_CENTER_STORE } from '@automattic/help-center/src/stores';
-import { Spinner } from '@wordpress/components';
+import {
+	useAttachFileToConversation,
+	useAuthenticateZendeskMessaging,
+} from '@automattic/zendesk-client';
+import { DropZone, Spinner } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
 import { useCallback, useRef, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import clsx from 'clsx';
-// eslint-disable-next-line no-restricted-imports
-import DropZone from 'calypso/components/drop-zone';
 import { SendMessageIcon } from '../../assets/send-message-icon';
 import { useOdieAssistantContext } from '../../context';
 import { useSendChatMessage } from '../../hooks';
 import { Message } from '../../types';
+import { zendeskMessageConverter } from '../../utils';
 import { AttachmentButton } from './attachment-button';
 import { ResizableTextarea } from './resizable-textarea';
 
 import './style.scss';
 
+const getFileType = ( file: File ) => {
+	if ( file.type.includes( 'image' ) ) {
```

Comment:
We should probably use `startsWith( 'image/' )`. [Some MIME types contain 'image' even though they're not images.](https://mimetype.io/all-types#:~:text=application/x%2Diso9660%2Dimage)

Reply:
wow great suggestion I missed this